acceptance_tests/features/pages/inbox_internal.py

```diff
@@ -35,3 +39,11 @@ def get_table_heading():
 
 def get_no_messages_text():
     return browser.find_by_text('No new messages')
+
+
+def get_ashe_radio_button_value():
+    return browser.find_by_id('ASHE')
+
```

Comment:
`get_radio_button_for_survey(survey):
          return browser.find_by_id(survey)
`
This might be better than having a function for each individual survey. Also function name is slightly misleading as it's not returning the value it's returning an element object.